src/components/admonition/examples/admonition-composite.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component, h, Prop, State } from '@stencil/core';
+
+/**
+ * Composite
+ *
+ * Show different types
+ */
+@Component({
+    tag: 'limel-example-admonition-composite',
+    shadow: true,
+})
+export class AdmonitionCompositeExample {
+    @Prop()
+    public schema: any;
+
+    @State()
+    private props: any = {
+        title: '',
+        description: 'This is my very nice [type]',
+        type: 'tip',
+    };
+
+    public componentWillLoad() {
+        const properties = { ...this.schema.properties };
+
+        this.schema = {
+            ...this.schema,
+            properties: properties,
+            lime: {
+                layout: {
+                    type: 'grid',
+                },
+            },
+        };
+    }
+
+    public render() {
+        const description = this.props?.description?.replace(
+            '[type]',
+            this.props?.type ?? ''
+        );
+
+        return [
+            <limel-admonition
+                {...{ ...this.props, description: description }}
+            />,
+            this.renderForm(),
+        ];
+    }
+
+    private renderForm() {
+        return (
+            <limel-example-controls
+                style={{ '--example-controls-column-layout': 'auto-fit' }}
```

Comment:
@Kiarokh I don't remember seeing this before. From the name alone, `auto-fit` sounds like a reasonable default though? What's the current default? 😊

Reply:
To follow the confusion tradition of CSS, these are just confusing names doing confusing things that is hard to guess or remember 😅 . Here are the docs: 

https://github.com/Lundalogik/lime-elements/blob/19465a9ca45f152790496f9449e35bd2a832a22d/src/examples/example-controls.scss#L3

But we have used it before in other examples and this usage seems to be correct.